src/lib/core/crash.c

```diff
@@ -392,32 +393,17 @@ crash_report_feedback_daemon(struct crash_info *cinfo)
 		[6] = NULL,
 	};
 
-	/*
-	 * Can't use fork here because libev has own
-	 * at_fork helpers with mutex where we might
-	 * stuck (see popen code).
-	 */
-	pid_t pid = vfork();
-	if (pid == 0) {
-		extern char **environ;
-		/*
-		 * The script must exit at the end but there
-		 * is no simple way to make sure from inside
-		 * of a signal crash handler. So just hope it
-		 * is running fine.
-		 */
-		execve(exec_argv[0], exec_argv, environ);
-		pr_crit("exec(%s,[%s,%s,%s,%s,%s,%s,%s]) failed",
-			exec_argv[0], exec_argv[0],
-			exec_argv[1], exec_argv[2],
-			exec_argv[3], exec_argv[4],
-			exec_argv[5], exec_argv[6]);
-		_exit(1);
-	} else if (pid < 0) {
-		pr_crit("unable to vfork (errno %d)", errno);
+#ifdef TARGET_OS_DARWIN
+	extern char **environ;
+#endif
```

Comment:
Nit: let's drop the ifdef and declare `envion` unconditionally, like it's done in other places, e.g. in [src/proc_title.c](https://github.com/tarantool/tarantool/blob/d9b8c0db79e7df4a024ca3273f663fd7d68a2d51/src/proc_title.c#L47)

Reply:
@locker the point is that, on Linux, `environ` is declared in unistd.h, and declaring it in source code on that platform is  redundant.

Reply:
But it wouldn't hurt, would it? Besides, it would fix the [build failure](https://github.com/tarantool/tarantool/runs/4427714455) on FreeBSD.

Reply:
@locker I think it's better to keep code platform-specific, i.e.
```c
#if defined(TARGET_OS_FREEBSD) || defined(TARGET_OS_NETBSD) || defined(TARGET_OS_DARWIN)
        extern char **environ;
#endif
```

But perhaps it's a matter of taste — just that my clang-tidy is complaining about a redundant declaration on the Linux platform.

Fixed: declared environ unconditionally.